organize/models.py

```diff
@@ -28,20 +35,144 @@ class EventApplication(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
 
     # application fields
-    about_you = models.TextField()
-    why = models.TextField()
-    involvement = models.CharField(choices=INVOLVEMENT_CHOICES, max_length=15)
-    experience = models.TextField()
-    venue = models.TextField()
-    sponsorship = models.TextField()
-    coaches = models.TextField()
+    about_you = models.TextField("About organizer")
+    why = models.TextField("Motivations to organize")
+    involvement = models.CharField("Involvement in Django Girls",
+                                   choices=INVOLVEMENT_CHOICES, max_length=15)
+    experience = models.TextField("Experience with organizing other events")
+    venue = models.TextField("Information about your potential venue")
+    sponsorship = models.TextField("Information about your potential sponsorship")
+    coaches = models.TextField("Information about your potential coaches")
+
+    # status reflecting state of the event in a triaging process.
+    status = models.CharField(
+        choices=APPLICATION_STATUS,
+        default=NEW,
+        max_length=10,
+    )
+    status_changed_at = models.DateTimeField(null=True, blank=True)
+
+    comment = models.TextField(null=True, blank=True)
 
     class Meta:
         permissions = (
             ("can_accept_organize_application",
              "Can accept Organize Applications"),
         )
 
+    def __str__(self):
+        return "{}, {} ({})".format(self.city, self.country, self.get_status_display())
+
+    def create_event(self):
```

Comment:
Methods on the model instance should basically modify it's state somehow or help retrieving it's values. Other things might as well be moved elsewhere. I feel all the methods related to creating the fresh event instance should just be put in a simple python file in one place in the core app. Now we have create_event here, add_default_content on the other instance. It's a bit messy and you don't know where to look for what. Also create_event changes the state of the Event model, not EventApplication model, and I would really really love to see it moved out from here 💃 🍡